### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Select Sunscriptions_Non-Text Contrast: Color contrast ratio for "Select Subscriptions" is 1.6:1 which is less than 3:1 when we select it.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2679]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### **GitHubTags:**
#Azure Tools for VS Code; #A11ySev2;  #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Non-Text Contrast; #WCAG1.4.11; #Win10;

**OS Details:**
Windows 10 Version 20H2 (OS Build 19042.685)

### **Environment Details:**
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab to Tree view and use down arrow to navigate to tree items.
8. Press enter on any tree view item
9. Verify Color contrast ratio for "Select Subscriptions" with background.

### Actual Result:
Color contrast ratio for "Select Subscriptions" is 1.6:1 which is less than 3:1 when we select it.

### Expected Result:
Color contrast ratio for "Select Subscriptions" with it's background should be more than 3:1

### User Impact:
Low vision users will find difficulty in viewing the content and to perform operations if color contrast ratio is not properly maintained

### Attachment for Reference:
![2679_Non text contarst issue](https://user-images.githubusercontent.com/38523250/108209168-fd3e3680-714f-11eb-8ae4-faf2d206a148.png)
